test/evergreen.yml

```diff
@@ -4321,6 +4321,34 @@ buildvariants:
     - name: cppsuite-search-near-02-stress
     - name: cppsuite-search-near-03-stress
 
+- name: cppsuite-stress-tests-arm64
+  display_name: "Cppsuite Stress Tests ARM64"
+  batchtime: 720 # twice a day
+  run_on:
+  - ubuntu2004-arm64-large
```

Comment:
Any special consideration here by using a large instance (instead of small as we normally do for test tasks)? If yes please add a comment.

Reply:
I think @luke-pearson tried and the medium variant does not exist?